modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/operation/InterceptingOperationMessageProcessorTestCase.java

```diff
@@ -16,19 +16,21 @@
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-
+import org.mule.runtime.core.api.Event;
```

Comment:
yes, since I'm wrapping the execution in the policy I need the policy mock to execute the real operation callback.